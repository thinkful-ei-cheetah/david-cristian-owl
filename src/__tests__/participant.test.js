import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import Participant from '../participant/participant';

describe('Participant component', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Participant name="Joe" inSession="true" onStage="false" avatar='https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1'/>, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders ui correctly', () => {
    const tree = renderer
    .create(<Participant name="Joe" inSession="true" onStage="false" avatar='https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1'/>)
    .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
