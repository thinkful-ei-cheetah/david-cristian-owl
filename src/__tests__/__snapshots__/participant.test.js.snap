// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Participant component renders ui correctly 1`] = `
<div
  className="person-container"
>
  <img
    alt="avatar"
    src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
  />
  <div>
    <h3>
      Joe
    </h3>
    <p
      style={
        Object {
          "color": "green",
        }
      }
    >
      in session
    </p>
  </div>
</div>
`;

exports[`renders ui correctly 1`] = `
<div
  className="person-container"
>
  <img
    alt="avatar"
    src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
  />
  <div>
    <h3>
      Joe
    </h3>
    <p
      style={
        Object {
          "color": "green",
        }
      }
    >
      in session
    </p>
  </div>
</div>
`;
